service/src/main/java/com/codeforcommunity/database/tableImpl/PostTable.java

```diff
@@ -0,0 +1,93 @@
+package com.codeforcommunity.database.tableImpl;
+
+import com.codeforcommunity.database.table.IPostTable;
+import com.codeforcommunity.dto.response.SinglePostResponse;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * Our implementation of the {@link IPostTable} in our database. This class will eventually be
+ * replaced once we start actually working with the database, but we'll still use it for testing
+ * purposes.
+ */
+public class PostTable implements IPostTable {
+  // Our in-memory database.
+  private final Map<Integer, SinglePostResponse> postMap;
+  // A count of how many posts we want to make on startup for the in memory db. You can change this
+  // to be whatever you want.
+  public static final int STUB_POST_COUNT = 15;
+  // The classic Lorem Ipsum text. For now this will be our stubbed data.
+  public static final String LOREM_IPSUM_TEXT =
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt "
+          + "ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation "
+          + "ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in "
+          + "reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. "
+          + "Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia "
+          + "deserunt mollit anim id est laborum.";
+
+  /** The constructor we'll be using in production. */
+  public PostTable() {
+    this.postMap = new HashMap<>();
+  }
+
+  /**
+   * The constructor we'll be using when we want the db to be seeded with fake data.
+   *
+   * @param postCount The count of posts to create
+   */
+  public PostTable(int postCount) {
```

Comment:
We may want to move all the random / stub data stuff into some different class. This is because the stub data is good for them to have for testing but when they actually run their app you'd want to just show the data that a user enters

Reply:
The `postCount` variable is so that can be controlled. I just wanted to have data initially so they can see something when they start up without having to add it. Once we start actually working with DBs, the current `tableImpl`s will be moved to the test modules so that they can be the mock databases.

Reply:
That's fine, I just think moving that code into a public static method in a different class will be cleaner than keeping it in the constructors